hera_cal/smooth_cal.py

```diff
@@ -825,20 +870,12 @@ def time_freq_2D_filter(self, freq_scale=10.0, time_scale=1800.0, tol=1e-09, fil
             win_kwargs : any keyword arguments for the window function selection in aipy.dsp.gen_window.
                 Currently, the only window that takes a kwarg is the tukey window with a alpha=0.5 default
         '''
-        # Generate DPSS filters if method='DPSS'
-        if method == 'DPSS':
-            design_matrix = dpss_filters(
-                freqs=self.freqs, times=self.time_grid, freq_scale=freq_scale, time_scale=time_scale,
-                eigenval_cutoff=eigenval_cutoff
-            )
-            # Keep track of the size of the weights grid after removing flagged edges
-            if skip_flagged_edges:
-                edges_old = None
-
-        else:
-            design_matrix = None
+        # Keep track of the size of the weights grid after removing flagged edges
+        if skip_flagged_edges:
+            edges_old = None
 
         # Create default variables to be passed in to time_freq_2D_filter
+        design_matrix = None
```

Comment:
I'd suggest explicitly initializing info as `{'sol_matrix': None, 'design_matrix': None}` so that the later calls are a bit less dependant on edges vs. edges_old and wgts_grid vs. wgts_old.

Reply:
Fixed this